### PR TITLE
Refactor filtering by attorney role

### PR DESCRIPTION
**Pull request by Zoe7**
The filter now works without having showing different tabs depending on the filter type. This means we can easily add new filters on top of the role, without having to add a new tab for each possible combination of the filters.

It also allows filtering by more than one role at the time (the buttons were swapped with checkboxes).

I don't think the result looks too good per se, since I was focusing on functionality and code. I'm super opened to any ideas on how to make it looks better (I tried inlining the checkboxes, but I feel like that makes it hard to parse the different options quickly). The whole thing would also look much nicer if we could animate some of these cards in and out, but honestly, I have zero clue how to do that with meteor/bootstrap 😄 

Here's a small gif of how it works/behaves:
ezgif com-gif-maker

### CODE REVIEW

**Comment by billimarie** [comment]
Wonderful work. Merging; I will create a design issue for the filter as well as the animation.

**Comment by billimarie** [comment]
@Zoe7 if you have a moment, can you take a look at why we're unable to click on the attorney card? It looks like the path for the attorneyView is missing, but I can't quickly figure out why.

**Comment by billimarie** [comment]
Hi @Zoe7, just a heads up: I had to revert the merge as I was unable to figure out why the route to the attorneyView was not working. I have an inkling it is due to the outdated Iron Router that we're using; I've created a separate issue for this (#127) but please feel free to resubmit this pull request if you are able to figure out why the card is not linking.

Thanks for all your hard work.

**Comment by Zoe7** [comment]
Ouch! Sorry, I didn't QA well enough 😞 
I found the issue. It's related to the fact that I changed the template to use the newer and recommended syntax for looping over the attorneys `{{#each ... in ...}}`. So we have to specify what data the router should use to generate the path.
I'll push the fix right away 😄

**Comment by billimarie** [comment]
No worries, my fault as well as I did not catch it in time :) Thank you for pushing the fix out so quickly. God aften, & hope to have you contribute in the future.
